src/PSRule.Rules.Azure/rules/Azure.KeyVault.Rule.ps1

```diff
@@ -124,3 +124,17 @@ Rule 'Azure.KeyVault.AutoRotationPolicy' -Ref 'AZR-000123' -Type 'Microsoft.KeyV
         );
     }
 }
+
+# Synopsis: KeyVaults should only accept explicitly allowed traffic.
+Rule 'Azure.KeyVault.Firewall' -Ref 'AZR-000355' -Type 'Microsoft.KeyVault/vaults'  -Tag @{ release = 'Preview'; 'Azure.WAF/pillar' = 'Security'; } {
```

Comment:
The release tag `Preview` is for preview Azure features. Since this is a generally available feature within Azure, instead of `Preview` use `GA`.

```suggestion
Rule 'Azure.KeyVault.Firewall' -Ref 'AZR-000355' -Type 'Microsoft.KeyVault/vaults'  -Tag @{ release = 'GA'; 'Azure.WAF/pillar' = 'Security'; } {
```